web/Web/web/search/templates/search/search_list.html

```diff
@@ -76,3 +76,29 @@ <h6 class="title" style="margin-bottom: 0;">{{ hit.title|safe }}</h6>
   </div>
   {% endif %}
 {% endif %}
+
+<nav aria-label="...">
```

Comment:
This block needs to be wrapped in a `{% if isQueryPage%}` to avoid `NoReverseMatch` error

e.g.
```
{% if isQueryPage %}
<nav ...>
...
</nav>
{% endif %>
```

Reply:
We're not using aria tags in this project, so no need to add it.

Reply:
Can you add `class="my-4"` to `<nav>` please. Everything else looks good!